server/upgrade.js

```diff
@@ -0,0 +1,70 @@
+const mongo   = require("promised-mongo")
+const db      = mongo("mongodb://localhost/bydesign", ["authors", "posts", "metadata", "categories"])
+const console = require("beautiful-log")
```

Comment:
Nothing wrong here for the time being, but just as an FYI, this wasn't the intended use of BL and also will not work in the next version of BL.